### DISCUSSION TITLE
Add scale/graph break feature to timeseries or graph panels

### DISCUSSION THREAD

**Discussion by chroberts** [grafana/grafana#61859] [Dashboards, Panels & Visualizations]
**What would you like to be added:**

I would like a feature for timeseries and/or graph (old) panels to have a graph break (or scale break) symbol -- see also [this thread discussing the name for this symbol](https://english.stackexchange.com/questions/196138/what-to-call-the-symbol-where-there-is-a-break-in-content-or-a-break-in-a-graph), which also has an example.

**Why is this needed:**

This symbol will help prevent [misleading graph](https://en.wikipedia.org/wiki/Misleading_graph) situations. Setting the y-axis to 0 can often help (and this is already supported), but it's sometimes also desired to be able to view broader trends or a zoomed in view, which the scale break symbol would allow.

There are many options for implementing this, e.g. to only support this feature for the axis or to also support individual data points when one or more are out of range, but I'm leaving this open ended for now.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82867. Feel free to continue the discussion around this there. Thank you!
